### ISSUE TITLE
jQuery dependency unfulfilled via jspm

### ISSUE THREAD

**Issue by Zoltu** [twbs/bootstrap#16233]
TL;DR:
Using JSPM to install bootstrap will not correctly bring in the jquery dependency.

Long:
I am using JSPM to do package management in a project.  I install bootstrap via JSPM using `jspm install bootstrap`.  This adds `"bootstrap": "github:twbs/bootstrap@^3.3.4",` to my `package.json` in the `jspm` section and `"bootstrap": "github:twbs/bootstrap@3.3.4",` to my `config.js` in the `map`.

Then in my .js file I do `import bootstrap from 'bootstrap'`.  At runtime, I receive an error complaining about being unable to load jquery.

I am pretty new to jspm and JavaScript development in general, but it seems like this package is just missing a jquery dependency.  With that, I believe JSPM will do the right thing and bring in jquery if I don't already have it.

**Comment by Zoltu** [comment]
This problem appears to have been introduced in 3.3.4.  If I install 3.3.2 with JSPM everything works.  Also of note, even after I got past the jQuery problem in 3.3.4 (manually installing it via JSPM), my project didn't work as expected.  I would have expected full backwards compatibility with 3.3.2 (not a major version update).  I did not continue investigating, so I am uncertain what exactly the problem was.

**Comment by cvrebert** [comment]
Approximate duplicate of #16101.
